src/ChargeCloudModels/ChargeCloudModels.jl

```diff
@@ -71,4 +71,48 @@ function create_charge_cloud(center::CartesianPoint{T}, energy::RealQuantity, pa
     end
     
     return NBodyChargeCloud{T, number_of_shells, typeof(shell_structure{T})}( points, energies./sum(energies) * T(to_internal_units(energy)), shell_structure{T})
+end
+
+function create_regular_sphere(center::CartesianPoint{T}, N::Integer, R::T)::Vector{CartesianPoint{T}} where {T <: SSDFloat}
+    
+    # Source: https://www.cmu.edu/biolphys/deserno/pdf/sphere_equi.pdf
+    
+    # There might be fluctuations of the order 10 points or 1% (whatever is larger)
+    points = Vector{CartesianPoint{T}}(undef, N + max(10, round(Int,1.01*N)))
+    Ncount = 0
+    a = 4π / N
+    Mθ = round(π/sqrt(a))
+    dθ = π/Mθ
+    for m in 0:Mθ-1
+        θ = π * (m + 0.5)/Mθ
+        Mφ = round(2π * sin(θ)*dθ/a)
+        for n in 0:Mφ-1
+            φ = 2π*n/Mφ
+            Ncount += 1
+            points[Ncount] = CartesianPoint{T}(R*cos(φ)*sin(θ), R*sin(φ)*sin(θ), R*cos(θ))
+        end
+    end
+    
+    return points[1:Ncount]
+end
+
+function create_charge_cloud(center::CartesianPoint{T}, energy::RealQuantity, N::Integer, particle_type::Type{PT} = Gamma;
```

Comment:
Now, when calling `create_charge_cloud` with an integer `N` after the energy, e.g. `create_charge_cloud(CartesianPoint{T}(0,0,0), 1460u"keV", 100)`, approximately `N` point-like charges are created on regular spheres. The number of spheres can be defined via `number_of_shells`.

For `N = 100`:

![regular_cloud_initial](https://user-images.githubusercontent.com/30291312/131902776-53cf150a-c4e4-470a-bae4-a3f327762227.png)

Creating an `Event` from the `NBodyChargeCloud` and drifting the charges with `diffusion` or `self-repulsion` leads a nice behavior. With this many point-like charges, I would not connect the points in the plot recipe.

Diffusion (`length = 5e-4`):
![regular_sphere_diffusion_5e-4](https://user-images.githubusercontent.com/30291312/131902940-d86e34b5-d0d0-4ef3-91eb-fbed632bdd9f.gif)

Self-repulsion:
![regular_sphere_self_repulsion](https://user-images.githubusercontent.com/30291312/131903017-b4d86edd-5b43-4a8f-aca1-20cdaf985d58.gif)

Diffusion (`length = 2e-4`) and self-repulsion:
![regular_sphere_both_2e-4](https://user-images.githubusercontent.com/30291312/131903135-05851054-2c63-4281-83ba-2e4b83d03167.gif)